frontend/uv-poc/src/app/customer-details/customer-details.component.html

```diff
@@ -8,6 +8,13 @@
       <th mat-header-cell *matHeaderCellDef class="text-right">Phone number</th>
       <td mat-cell *matCellDef="let details" class="text-right">
         {{ details.phoneNo }}
+        <mat-icon
+          aria-hidden="false"
+          class="icon-display"
+          aria-label="Example home icon"
```

Comment:
This isn't exactly what we want to see when hovering over the icon `Example home icon`
Please come up with something better